src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Decompression.cs

```diff
@@ -43,6 +51,15 @@ public static IEnumerable<object[]> DecompressedResponse_MethodSpecified_Decompr
         public async Task DecompressedResponse_MethodSpecified_DecompressedContentReturned(
             string encodingName, Func<Stream, Stream> compress, DecompressionMethods methods)
         {
+            _output.WriteLine($"encodingName={encodingName}");
+            _output.WriteLine($"methods={methods}");
```

Comment:
I would consider moving these the after the check below, to keep it clear that those cases are not run on `WinHttpHandler` or `CurlHandler`

Reply:
I'll make that change.